### PR TITLE
Add manual testing for command suggestions

### PR DESCRIPTION
**Pull request by cutieprobe**
- Add manual testing instructions for command suggestion and auto-completion feature.
- Fix minor bugs in  edit and find command prompters.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -1206,6 +1206,100 @@ Expected: No meme is liked. Error details shown in the status message. Status ba
...
+    Expected: +
+    `add: adds a meme to Weme. +
+    archive: archive a meme by index. +
+    archives: list all archived memes.`
+..  Test case: `a` + kbd:[tab] +
+    Expected text inside command box: `add{nbsp}` +
+    Expected text inside result box: +
+    `add: adds a meme to Weme. Parameters: p/PATH [d/DESCRIPTION] [t/TAG]... +
+Example: add p/C:\Users\username\Downloads\funny_meme.jpg d/Popular Meme among SoC Students  t/funny`
+..  Test case: `arkive` +
```
Comment (moziliar) [inline docs/DeveloperGuide.adoc]: The typo is on purpose?

Comment (cutieprobe) [reply docs/DeveloperGuide.adoc]: Yes, to show that typo can be auto-corrected by suggestion.

File: src/main/java/seedu/weme/logic/prompter/commandprompter/createcommandprompter/TextEditCommandPrompter.java [hunk 2]
```diff
@@ -39,8 +42,8 @@ public CommandPrompt prompt(Model model, String userInput) throws PromptExceptio
             return new CommandPrompt(MESSAGE_USAGE, userInput);
         }
 
-        ArgumentTokenizer.tokenize(arguments, PREFIX_TEXT, PREFIX_X_COORDINATE, PREFIX_Y_COORDINATE,
-                PREFIX_COLOR, PREFIX_STYLE, PREFIX_SIZE);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(arguments, PREFIX_TEXT, PREFIX_X_COORDINATE,
```
Comment (moziliar) [inline src/main/java/seedu/weme/logic/prompter/commandprompter/createcommandprompter/TextEditCommandPrompter.java]: Is argMultimap used anywhere?

Comment (cutieprobe) [reply src/main/java/seedu/weme/logic/prompter/commandprompter/createcommandprompter/TextEditCommandPrompter.java]: Line 70

**Pull request merged**